packages/@o3r/schematics/src/rule-factories/updates-of-old-otter-scope/imports/list-of-vars.ts

```diff
@@ -1,4 +1,19 @@
-export const listOfExposedElements: {value: string; type: 'function' | 'var'; replacement?: string}[] = [
+/**
+ *
+ * Represents an element that is exposed in a Sass import.
+ *
+ * @interface
+ * @property {string} value - The name of the exposed element.
```

Comment:
This is a type script interface, no need to rue-specify the type in JS. It is actually not a good practice